easybuild/easyconfigs/r/RELION/RELION-2.1-foss-2018a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '2.1'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/3dem/relion/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['08729aac4fa0841e08df2c90a2c26cfe2aef52af2f5fb94a770ed2c11373b886']
+
+builddependencies = [('CMake', '3.10.1')]
+
+dependencies = [
+    ('X11', '20180131'),
+    ('FLTK', '1.3.4'),
+]
+
+configopts = "-DCMAKE_SHARED_LINKER_FLAGS='-lpthread' "
+
+sanity_check_paths = {
+    'files': ['bin/relion', 'lib/librelion_gui_lib.so', 'lib/librelion_lib.so'],
```

Comment:
@smoors Please use `SHLIB_EXT` rather than hardcoding `.so`:

```python
    'files': ['bin/relion', 'lib/librelion_gui_lib.%s' % SHLIB_EXT, 'lib/librelion_lib.%s' % SHLIB_EXT],

Reply:
I always forget that one..